code/modules/food_and_drinks/food/foods/side_dishes.dm

```diff
@@ -117,6 +117,16 @@
 	tastes = list("rice" = 1)
 	goal_difficulty = FOOD_GOAL_NORMAL
 
+/obj/item/food/snacks/boiledrice/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/stack/seaweed))
+		return ..()
+	var/obj/item/stack/seaweed/S = I
+	var/obj/item/food/snacks/onigiri/O = new
+	S.use(1)
```

Comment:
```suggestion
	if(!istype(I, /obj/item/stack/seaweed))
		return ..()
	var/obj/item/stack/seaweed/S = I
	if(!S.use(1))
		return ..()
	var/obj/item/food/snacks/onigiri/O = new(get_turf(user))
```
Always put your `use(x)` in an `if`-statement, it'll return `FALSE` if for some reason you cannot use 1 of the stack!